releasenotes/notes/bumps-redfish-exporter-to-v2-11032fb9dde36283.yaml

```diff
@@ -0,0 +1,19 @@
+---
+features:
+  - Upgrades the redfish exporter container image to the v2.x series.
+  - Adds support for lenovo hardware to the redfish exporter dashboard.
+  - |
+    Adds the ``stackhpcredfish_exporter_scrape_interval``,
+    ``stackhpc_os_capacity_scrape_interval``, and
+    ``stackhpc_prometheus_openstack_exporter_interval``
+    configuration variables.
+fixes:
+  - |
+    Sets the prometheus server side option ``query.lookback-delta`` to
+    the largest scrape interval so that metrics are not from exporters
+    with large scrape intervals are not marked stale before the next scrape.
```

Comment:
```suggestion
  - |
    Sets the prometheus server side option ``query.lookback-delta`` to
    the largest scrape interval so that metrics from exporters with large
    scrape intervals are not marked stale before the next scrape.
```